AdyenCard/Form/FormCardNumberContainerItem.swift

```diff
@@ -71,22 +69,32 @@ internal final class FormCardLogosItem: FormItem, Hidable {
     
     internal var subitems: [FormItem] = []
     
-    internal let cardLogos: [CardTypeLogo]
-    
     internal let style: FormTextItemStyle
     
-    /// Observable property to update the owner view's alpha.
-    @AdyenObservable(1) internal var alpha: CGFloat
+    @AdyenObservable([]) internal var cardLogos: [CardTypeLogo]
     
     internal init(cardLogos: [CardTypeLogo], style: FormTextItemStyle) {
-        self.cardLogos = cardLogos
         self.style = style
```

Comment:
why are we deleting and adding "self.cardLogos = cardLogos" ?

Reply:
It's just changing the order, not deleting. Now the `cardLogos` has a default value, all properties without a value must be set first in the init in order to access self so I moved it below.

Reply:
make sense :)